### PR TITLE
Improving most ultimates

### PR DESCRIPTION
**Pull request by POCKETMONEY99**
Based on Issue  #2681. If you want more details about why I have done this read this issue. 
Tl;dr version. Rescaling most ultimates so they don't suck in the midgame (LVL 18-37). 
I am not certain on all my changes, but I am certain on most of them. Some of them I did for consistency reasons. Please comment on any changes you don't like.

### CODE REVIEW

**Comment by Larenall** [comment]
I like most of the changes. One exeption is leshrac. Now aghs boosts leshrac ultimate too much in early game. Dont forget that he has a lvl20 talent that gives +80 pulse nova dmg iirc. Honestly even with current numbers he is a good hero and im afraid that its an overbuff. I do believe he also deserves a buff, but not as big.

File: game/scripts/npc/abilities/enigma_black_hole.txt [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "200.0 180.0 160.0 145.0 130.0"
+    "AbilityCooldown"                                     "140.0" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/enigma_black_hole.txt]: AAAAAAAAAAA for the love of god. Set the cooldown to 160 at all levels or don't change it at all.

File: game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt [hunk 2]
```diff
@@ -43,7 +43,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage"                                          "110 140 170 340 510"
+        "damage"                                          "110 170 230 370 510" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt]: How is the scaling of wrath of nature damage with scepter after this change?

File: game/scripts/npc/abilities/legion_commander_duel.txt [hunk 3]
```diff
@@ -27,7 +27,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "50.0"
+    "AbilityCooldown"                                     "35.0" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/legion_commander_duel.txt]: Ah here we go again. I give up.

File: game/scripts/npc/abilities/leshrac_pulse_nova.txt [hunk 4]
```diff
@@ -45,13 +45,13 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage"                                          "100 140 180 360 580"
+        "damage"                                          "100 180 260 420 580" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/leshrac_pulse_nova.txt]: Buff that cancer more, MORREEEEE

File: game/scripts/npc/abilities/leshrac_pulse_nova.txt [hunk 5]
```diff
@@ -45,13 +45,13 @@
...
         "LinkedSpecialBonus"  "special_bonus_unique_leshrac_3"
       }
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage_scepter"                                  "160 190 220 450 680"
+        "damage_scepter"                                  "200 280 360 520 680" //OAA, Old Aghs
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/leshrac_pulse_nova.txt]: You know that this does nothing, right? This KeyValue is for nothing. Valve just forgot to remove it.

File: game/scripts/npc/abilities/magnataur_reverse_polarity.txt [hunk 6]
```diff
@@ -18,7 +18,7 @@
     // Stats
     //-------------------------------------------------------------------------------------------------------------
     "AbilityCastPoint"                                    "0.3"
-    "AbilityCooldown"                                     "130"
+    "AbilityCooldown"                                     "100" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/magnataur_reverse_polarity.txt]: Cooldown for this ability is perfectly fine. But I am fighting windmills.

File: game/scripts/npc/abilities/omniknight_guardian_angel.txt [hunk 7]
```diff
@@ -18,7 +18,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "160" //OAA
+    "AbilityCooldown"                                     "120" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/omniknight_guardian_angel.txt]: The cooldown for this spell is perfectly fine. Don't change this plz.

File: game/scripts/npc/abilities/pangolier_gyroshell.txt [hunk 8]
```diff
@@ -33,7 +33,7 @@
 
     // Damage.
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityDamage"                 "200 275 350 550 800"
+    "AbilityDamage"                 "200 350 500 750 1000" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/pangolier_gyroshell.txt]: Buff Larenall more.

File: game/scripts/npc/abilities/silencer_global_silence.txt [hunk 9]
```diff
@@ -24,7 +24,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "130 120 110 100 90" //OAA
+    "AbilityCooldown"                                     "100" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/silencer_global_silence.txt]: Last level was meant to be 75 seconds at one point. Use this information however you want.

File: game/scripts/npc/abilities/slark_shadow_dance.txt [hunk 10]
```diff
@@ -20,7 +20,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "80 70 60 55 50"
+    "AbilityCooldown"                                     "50 45 40 35 30" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/slark_shadow_dance.txt]: Too much. Put `60 55 50 45 40`

File: game/scripts/npc/abilities/slark_shadow_dance.txt [hunk 11]
```diff
@@ -70,7 +70,7 @@
       "08"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "cooldown_scepter"                                "30"
+        "cooldown_scepter"                                "25" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/slark_shadow_dance.txt]: Revert this. Not needed. Also meaningless.

File: game/scripts/npc/abilities/tidehunter_ravage.txt [hunk 12]
```diff
@@ -26,11 +26,11 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "150.0"
+    "AbilityCooldown"                                     "130.0" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/tidehunter_ravage.txt]: You have a seperate pull request for this as far as I know.

File: game/scripts/npc/abilities/zuus_thundergods_wrath.txt [hunk 13]
```diff
@@ -56,7 +56,7 @@
       "06"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage"                                          "225 325 425 725 1025"
+        "damage"                                          "225 425 625 1025 1425" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/zuus_thundergods_wrath.txt]: Tone down OAA levels.

**Comment by DarkoniusXNG** [comment]
Btw sorry for being rude in some comments. I lost my shit reading some of this changes.

**Comment by Larenall** [comment]
> I like most of the changes. One exeption is leshrac. Now aghs boosts leshrac ultimate too much in early game. Dont forget that he has a lvl20 talent that gives +80 pulse nova dmg iirc. Honestly even with current numbers he is a good hero and im afraid that its an overbuff. I do believe he also deserves a buff, but not as big.

I forgot scepter doesnt give dmg LUL.

**Pull request closed without merging**